app/src/main/java/fi/aalto/legroup/achso/entities/PooledVideo.java

```diff
@@ -0,0 +1,132 @@
+package fi.aalto.legroup.achso.entities;
+
+import android.graphics.PointF;
+import android.location.Location;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Holds a Video object and a persistent list of Annotations.
+ *
+ * You can request Video objects with specified amount of annotations from this and it will set
+ * the annotations to be a sub-list of the persistent annotation list, so there is no need to
+ * allocate annotations if it has enough storage for them. Otherwise it will allocate more
+ * storage.
+ */
+public class PooledVideo {
+
+    private Video video;
+    private ArrayList<Annotation> annotations;
+    private Location location;
+    private boolean inUse;
+
+    private static final int defaultAnnotationCount = 32;
+    private static final boolean defaultHasLocation = true;
```

Comment:
The rest of the code uses `SCREAMING_SNAKE_CASE` for constants, could you change these to use that as well for consistency?